tests/transformers/searchservice/createDirectAnswer.ts

```diff
@@ -1,5 +1,7 @@
 import { createDirectAnswer } from '../../../src/transformers/searchservice/createDirectAnswer';
-import { DirectAnswerType } from '../../../src/models/searchservice/response/DirectAnswerType';
+import { DirectAnswerType } from '../../../src/models/searchservice/response/directanswer/DirectAnswerType';
+import { CustomFieldValueDA } from '../../../src/models/searchservice/response/directanswer/CustomFieldValueDA';
```

Comment:
I thought the linting check was supposed to fail even on warnings like for this unused import?

<img width="634" alt="Screen Shot 2022-08-12 at 2 44 21 PM" src="https://user-images.githubusercontent.com/88398086/184423570-40fba010-65bb-4729-848d-8835a97e92f6.png">

Reply:
looks like we have `--max-warnings=0` in the lint script command so it should fail I think. not sure why

Reply:
we don't have --max-warnings=0 set in the lint command here

Reply:
my bad, looked at the wrong repo. I can add that in